.gitignore

```diff
@@ -8,4 +8,5 @@ dependency-reduced-pom.xml
 .project
 .settings/
 .vscode
-Test.java
\ No newline at end of file
+Test.java
+src/main/java/cl/transbank/webpay/example/controller/borrar
```

Comment:
This should be erased, the name says so 😛

Reply:
In fact it is to do not track files in this path because I use it for some local testing